dual_net.py

```diff
@@ -320,7 +321,8 @@ def validate(working_dir, tf_records, checkpoint_name=None, **hparams):
     estimator = get_estimator(working_dir, **hparams)
     if checkpoint_name is None:
         checkpoint_name = estimator.latest_checkpoint()
-    input_fn = lambda: preprocessing.get_input_tensors(
+
+    def input_fn(): return preprocessing.get_input_tensors(
```

Comment:
huh, these got changed from lambdas to def's -- are the autopep8 flags different?